free-programming-books-id.md

```diff
@@ -18,6 +23,10 @@
 * [Tutorial Membuat Aplikasi Galeri Foto Android](https://www.smashwords.com/books/view/533096)
 
 
+### <a id="cplusplus"></a> C++
+
+* [Koding C++ Dengan Qt](https://leanpub.com/koding-cpp-qt)
```

Comment:
we add an access note for leanpub depending on whether the book content can be viewed without registration